### PR TITLE
new version of docker client with embedded commons codec

### PR DESCRIPTION
**Pull request by danushkaf**

### CODE REVIEW

**Comment by danushkaf** [comment]
We are on the process of migrating app cloud application to AS 5.3.0. While doing that we got osgi dependency issue in AS 5.3.0. 
Problem is the following bundles ship with AS 5.3.0 use conflicts packages.
opensaml 2.4.1.wso2v1 uses commons.codec.binary. But pom.xml use dynamic imports
sso.common 4.5.5 uses common.codec.binary. pom.xml use range 1.4.0 - 1.5.0
docker client 1.0.10 uses commons.codec 1.10.0
AS 5.3.0 ship commons.codec 1.4.0

We need to deploy commons.codec 1.10.0. But when we do that opensaml binds to new version and sso.common binds to old version and server doesn't statup. We tried several possible solutions (try installing new feature to AS 5.3.0, Try updating opensaml orbit to use old versions, etc..).

But since only successful solution was to embed commons codec to docker client orbit and couldn't find any other option, we will go ahead with this and will send a PR for new docker client orbit version.

P.S. 
How this works in AS 5.2.1 ?
AS 5.2.1 packs sso.common 4.2.0 which dont use a range of commons.codec.binary. So both bundles binds to new version. So it works without an issue.

P.S. 
How this works in AS 5.2.1 ?
AS 5.2.1 packs sso.common 4.2.0 which dont use a range of commons.codec.binary. So both bundles binds to new version. So it works without an issue.

File: docker-client/1.0.10.wso2v2/pom.xml [hunk 1]
```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ ~ Copyright (c) 2005-2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (laki88) [inline docker-client/1.0.10.wso2v2/pom.xml]: licensee header is not correct. it should not contain year range

Comment (danushkaf) [inline docker-client/1.0.10.wso2v2/pom.xml]: Fixed

File: docker-client/1.0.10.wso2v2/pom.xml [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    </repositories>
+
+    <distributionManagement>
+        <repository>
+            <id>wso2.releases</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/releases/</url>
+        </repository>
+
+        <snapshotRepository>
```
Comment (thusithathilina) [inline docker-client/1.0.10.wso2v2/pom.xml]: no need of snapshot repo

**Pull request merged**